pkg/rke2/rke2.go

```diff
@@ -32,10 +32,20 @@ func Server(clx *cli.Context, cfg Config) error {
 	if err := setup(clx, cfg); err != nil {
 		return err
 	}
+
 	if err := clx.Set("secrets-encryption", "true"); err != nil {
 		return err
 	}
 
+	// Disable all disableable k3s packaged components. In addition to manifests,
+	// this also disables several integrated controllers.
```

Comment:
Which controllers?

Reply:
ServiceLB controller is disabled here:
https://github.com/rancher/k3s/blob/master/pkg/cli/server/server.go#L194
Node controller is disabled here:
https://github.com/rancher/k3s/blob/master/pkg/server/server.go#L165

The Cloud Controller frustratingly does the opposite - it has a dedicated `--disable-cloud-controller` flag which adds a hidden entry to the --disable list to skip deploying the RBAC manifest.